theatre/core/src/sheetObjects/SheetObjectTemplate.ts

```diff
@@ -138,8 +143,7 @@ export default class SheetObjectTemplate {
 
           const propConfig = getPropConfigByPath(this.config, pathToProp)
 
-          if (!propConfig || !propConfig?.sanitize || !propConfig.interpolate)
-            continue
+          if (!propConfig || !isPropConfSequencable(propConfig)) continue
```

Comment:
Nit: Consider pulling this out into a named variable like the following to reduce the negations.
```ts
const propConfig = getPropConfigByPath(this.config, pathToProp)
const propIsSequenceable = propConfig && isPropConfSequencable(propConfig)
if (!propIsSequenceable) continue
```

Reply:
Fixed here 679629def80d52e8fe48a12e90463d49d81d4bd4